modules/storage/cosmos-db/.bicep/nested_rbac.bicep

```diff
@@ -0,0 +1,39 @@
+param description string = ''
+param principalIds array
+param principalType string = ''
+param roleDefinitionIdOrName string
+param resourceId string
```

Comment:
i prefer to use the name of the resource instead of the resource id to create existing resources.

Reply:
changes parameter to `cosmosDBAccountName`.